docs/how-to/build/eps.md

```diff
@@ -87,18 +87,18 @@ If you want to build with an earlier version, you must temporarily remove the 'C
 
 ## TensorRT
 
-See more information on the TensorRT Execution Provider [here](../reference/execution-providers/TensorRT-ExecutionProvider.md).
+See more information on the TensorRT Execution Provider [here](../../reference/execution-providers/TensorRT-ExecutionProvider.md).
 
 ### Prerequisites
 {: .no_toc }
 
 * Install [CUDA](https://developer.nvidia.com/cuda-toolkit) and [cuDNN](https://developer.nvidia.com/cudnn)
-   * The TensorRT execution provider for ONNX Runtime is built and tested with CUDA 11.0 and cuDNN 8.0.
+   * The TensorRT execution provider for ONNX Runtime is built and tested with CUDA 11.1 and cuDNN 8.2.
```

Comment:
i think we need to change this to a support matrix (table)
where we indicate that we also support other versions of TRT/CUDA/CuDNN
e.g. 7.1.3.4/10.2/7.6.x is also valid combination